### DISCUSSION TITLE
Need help - deploying spa project to azure static web apps

### DISCUSSION THREAD

**Discussion by ecmap** [quasarframework/quasar#10409] [General - Components / Directives / etc] [answered]
This is what my github project looks like (quasar 2.0.x vue3)

![螢幕快照 2021-08-17 下午6 21 34](https://user-images.githubusercontent.com/63713926/129710212-4e628aa6-d841-4978-80e6-fdcb96ac67a9.png)

This is how I set up a static web apps

![螢幕快照 2021-08-17 下午6 23 03](https://user-images.githubusercontent.com/63713926/129710400-373ab533-6ff3-4aab-8c31-45ba6a33d334.png)

But I encountered the following error during github Action, how to solve it :)?

![螢幕快照 2021-08-17 下午6 24 41](https://user-images.githubusercontent.com/63713926/129710630-72176e19-4388-4e92-9ed6-9142aa90bffc.png)
![螢幕快照 2021-08-17 下午6 24 26](https://user-images.githubusercontent.com/63713926/129710674-9e8e24e5-12ed-459e-9f11-05844b4ea260.png)

**Comment by MilosPaunovic** [comment, marked as answer]
Do you have `bulid` script in you `package.json`?

**Reply by ecmap** [reply]
> Do you have `bulid` script in you `package.json`?

my package.json

![螢幕快照 2021-08-17 下午6 47 05](https://user-images.githubusercontent.com/63713926/129713216-12e65494-8180-4aad-9c1d-2840f1dfc207.png)
![螢幕快照 2021-08-17 下午6 47 32](https://user-images.githubusercontent.com/63713926/129713229-b6165c69-0acf-43eb-93b2-73f2608229c3.png)

I am new to azure, even though I have read a lot of tutorials (vue / react or other deployment), I don't know what I should put or do in my project

update: after adding build: quasar build 
solve the problem, @MilosPaunovic thank you save my life

**Reply by MilosPaunovic** [reply]
This has nothing to do with Azure. You need to build your Quasar project first.

Check this out for help with the build process.
https://quasar.dev/quasar-cli/build-commands

**Reply by MilosPaunovic** [reply]
Okay, if you solved it, great 😄
